kolibri/core/error_reports/middleware.py

```diff
@@ -72,13 +76,13 @@ def process_exception(self, request, exception):
             "request_info": get_request_info(request),
             "server": get_server_info(request),
             "packages": get_packages(),
-            "python_version": get_python_version(),
+            "python_version": ".".join(str(v) for v in version_info[:3]),
```

Comment:
Maybe you missed this one, should call the method instead as it is also mocked in test_middleware

Reply:
Hrm, weird - I patched that, but I guess it didn't take!

Reply:
Updated!